src/internal.c

```diff
@@ -14904,6 +14909,15 @@ static word32 QSH_KeyExchangeWrite(WOLFSSL* ssl, byte isServer)
                         return NO_PEER_KEY;
                     }
 
+                #ifdef HAVE_PK_CALLBACKS
+                    if (ssl->ctx->EccSharedSecretCb) {
+                        ret = ssl->ctx->EccSharedSecretCb(ssl, es + 1, &pubKeySz, 
```

Comment:
This return code (ret) looks like it will be lost by later call to wc_ecc_export_x963() 17 lines later.  Should probably check here.